spec/models/client_spec.rb

```diff
@@ -49,7 +49,9 @@ def app
     client = Client.create(name: 'Delete Me')
     expect(client).to exist
     expect { client.delete }.to_not raise_error
+    ### Test passes but the item's name and address can still be printed?
     expect(client).to_not exist
+    print client, ' ', client.name
```

Comment:
Is ``expect(client).to_not exist`` logically equivalent to ``expect(client.exists?).to be false``?